apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/server/RandomFissClaimSource.java

```diff
@@ -67,7 +67,7 @@ public FissClaimChange next() {
 
     final Timestamp timestamp =
         Timestamp.newBuilder().setSeconds(clock.instant().getEpochSecond()).build();
-    final var claim = generator.randomClaim();
+    final var claim = generator.randomClaim(); // here
```

Comment:
I think we can revert this
```suggestion
    final var claim = generator.randomClaim();
```